tests/integration/aof.tcl

```diff
@@ -655,4 +655,20 @@ tags {"aof external:skip"} {
             }
         }
     }
+
+    start_server {} {
+        test {FLUSHDB / FLUSHALL should persist in AOF} {
+            r config set appendonly no
+            r multi
+            r config set appendonly yes
+            r config set appendonly no
+            r exec
+
+            r config set appendonly yes
+            r multi
+            r config set appendonly no
+            r config set appendonly yes
+            r exec
```

Comment:
No need to check the reply of theae commands? Or we only need to check that we don't crash?

Please add some comment about it.

Reply:
i just want a test that can cover this case, i added a comment